pandas/core/arrays/interval.py

```diff
@@ -648,7 +648,7 @@ def fillna(self, value=None, method=None, limit=None):
             )
             raise TypeError(msg)
 
-        value = getattr(value, "_values", value)
+        value = extract_array(value, extract_numpy=True)
```

Comment:
There is a ` if not isinstance(value, Interval)` just above, meaning that `value` should be an Interval, and this line can just be removed altogether?

Reply:
good catch, will update